lib/Listener/NewMessageClassificationListener.php

```diff
@@ -82,6 +96,7 @@ public function handle(Event $event): void {
 			foreach ($event->getMessages() as $message) {
 				if ($predictions[$message->getUid()] ?? false) {
 					$message->setFlagImportant(true);
+					$message->setTags([$important]);
```

Comment:
:warning:  this will overwrite any other existing tags

add a `addTag` that appends